### ISSUE TITLE
7.5.1804 should be tagged for AAarch64 (ARM64)

### ISSUE THREAD

**Issue by lag-linaro** [CentOS/sig-cloud-instance-images#131]
When `docker pull`ing `centos:7` on an AArch64 platform you get `7.5.1804`.

But if `centos:7.5.1804` is `docker pull`ed a message is displayed saying that it's not supported.

Could someone correct the [CentOS Docker Library](https://github.com/docker-library/official-images/blob/master/library/centos) to explicitly enable AArch64 for `7.5.1804` please?

**Comment by lag-linaro** [comment]
When looking at the library file it appears as though you are missing `Architecture:` tags from all of your specific releases.  Would anyone care to address that such that !x86_64 consumers and reference them directly?

**Comment by lag-linaro** [comment]
@jperrin are you following this?

**Comment by jperrin** [comment]
We didn't officially add aarch64 support for docker images until after 7.5.1804, even though we'd published them for a while. Currently it's only tagged for 7. 

We can fix this for 7.6.1810 when we release it.

**Comment by lag-linaro** [comment]
Right, but (7 == 7.6.1804):
```
$ docker run -ti --rm centos:7
[root@84d27e1dfa04 /]# cat /etc/centos-release
CentOS Linux release 7.5.1804 (AltArch)
```
So sounds fair to tag `7.5.1804` for AArch64 support, right?

**Comment by jperrin** [comment]
No, the minor tags are designed to *only* match the install media. They don't change once they're created.

**Comment by lag-linaro** [comment]
Would you be kind enough to expand on that please?  I'm not sure I understand exactly what you mean.
